tl.tl

```diff
@@ -1837,6 +1853,9 @@ do
       i, lhs = P(ps, i)
       i, lhs = E(ps, i, lhs, 0)
       if lhs then
+         if node_is_require_call(lhs) then
+            table.insert(ps.require_calls, lhs)
```

Comment:
instead of extending Node, `require_calls` could be a `{string}` and just store the contents of the literal (`lhs.e2[1].tk`)